launch_testing/launch_testing/asserts/assert_exit_codes.py

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+A module providing exit code assertions.
+
+PYTEST_DONT_REWRITE
```

Comment:
Maybe, add a comment/link explaining why this is needed (e.g.: https://docs.pytest.org/en/latest/assert.html#assertion-introspection-details).

Reply:
Sure, done in c7e6f8043a9643ac2f3a6f2206c7186011dff32e.